edb/pgsql/metaschema.py

```diff
@@ -185,6 +185,106 @@ def __init__(self) -> None:
         )
 
 
+class StrToInt64NoInline(dbops.Function):
+    """String-to-int64 cast with noinline guard."""
+    text = r'''
+        SELECT
+            "val"::bigint
+        LIMIT
```

Comment:
Do you know why LIMIT 1 is non-ininable? Maybe you can document that?

Reply:
It's probably not "limit 1". It probably nested function call that is not inlinable. But I'm not sure.

Reply:
No, it's `LIMIT` specifically: https://github.com/postgres/postgres/blob/24d2d38b1eb86c0b410ad0f07f66566a83c6f05c/src/backend/optimizer/util/clauses.c#L4501-L4529

I added a comment to clarify.